README.md

```diff
@@ -192,6 +260,30 @@ http://localhost:5601
 In order to show documents in the case-server index with Kibana, you must first create the index pattern ('Management' page) : case-server*
 
 
+### Installing / Updating docker-compose to v2
+Docker-compose v2 is necessary to have to profiles feature.  
+If possible, prefer to install it with your package manager if you are on a Unix system.
+
+> [!NOTE]  
+> You need a client (docker-cli) of ~v19~ v20 at least to have the system of cli-plugins.
+> It isn't necessary to update your Docker engine or client else.
+
+You can install docker-compose with these commands, as instructed in [the doc](https://docs.docker.com/compose/install/linux/#install-the-plugin-manually) and the [migration guide](https://docs.docker.com/compose/migrate/):
+```shell
+curl -LR --create-dirs -o $HOME/.local/bin/docker-compose https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64
+#wget -x -O $HOME/.local/bin/docker-compose https://github.com/docker/compose/releases/latest/download/docker-compose-linux-x86_64
+chmod +x $HOME/.local/bin/docker-compose
+mkdir -p $HOME/.docker/cli-plugins
+ln -s $HOME/.local/bin/docker-compose $HOME/.docker/cli-plugins/docker-compose
+docker compose version
+```
+You must get this output from docker compose now:
+> Docker Compose version v2.20.3
```

Comment:
```suggestion
> Docker Compose version v2.X.X
```

Reply:
That is just an example of the output, i don't think it's necessary... no?

Reply:
For me it's better, otherwise we can think that we need this version specifically